nominees/akvo-flow.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Akvo FLOW",
+  "description": "Akvo Flow is a tool for collecting, evaluating and displaying of geographically referenced data. It is composed of an android mobile app and an online web-based platform.",
+  "website": "https://akvo.org/flow-caddisfly-lumen/",
```

Comment:
@rachita19082 could you capitalize only the first letter of the word "FLOW" for the name field to maintain consistency with the website. Thanks.

Reply:
Resolved in commit bea69e8